apps/web-mzima-client/src/app/settings/surveys/surveys.component.ts

```diff
@@ -1,11 +1,12 @@
 import { Component, OnInit } from '@angular/core';
-import { MatCheckboxChange } from '@angular/material/checkbox';
+// import { MatCheckboxChange } from '@angular/material/checkbox';
 import { TranslateService } from '@ngx-translate/core';
```

Comment:
If a piece of code is not needed, remove it rather than commenting it out :).

Reply:
True.. thanks for the feedback, I'll remove right away